src/main/java/org/snomed/snowstorm/core/data/services/BranchReviewService.java

```diff
@@ -408,73 +409,149 @@ private void lookupBranchReviewConceptChanges(BranchReview branchReview) {
 		start.setTime(start.getTime() + 1);
 
 		Set<Long> changedConcepts = createConceptChangeReportOnBranchForTimeRange(source.getPath(), start, source.getHead(), branchReview.isSourceParent());
-		changedConcepts.addAll(getContradictoryConceptIdentifiers(branchReview, source, target));
+		changedConcepts.addAll(getConceptIdentifiersDeletedSomewhereWithDescriptionsModifiedElsewhere(branchReview, source, target));
 		branchReview.setStatus(ReviewStatus.CURRENT);
 		branchReview.setChangedConcepts(changedConcepts);
 		branchReviewRepository.save(branchReview);
 	}
 
+	private Set<Long> getConceptIdentifiersDeletedSomewhereWithDescriptionsModifiedElsewhere(BranchReview branchReview, Branch source, Branch target) {
+		String sourcePath = source.getPath();
+		String targetPath = target.getPath();
+
+		Set<String> conceptIdsWithModifiedDescriptionsOnSource = getConceptIdentifiersWithModifiedDescriptionsOnSource(branchReview, source, target);
+		if (conceptIdsWithModifiedDescriptionsOnSource.isEmpty()) {
+			return Collections.emptySet();
+		}
+
+		Set<Long> conceptIdentifiers = new HashSet<>();
+		conceptIdentifiers.addAll(getConceptIdentifiersDeletedOnSource(conceptIdsWithModifiedDescriptionsOnSource, source, target));
+		conceptIdentifiers.addAll(getConceptIdentifiersDeletedOnTarget(conceptIdsWithModifiedDescriptionsOnSource, source, target));
+
+		return conceptIdentifiers;
+	}
+
+	private Set<String> getConceptIdentifiersWithModifiedDescriptionsOnSource(BranchReview branchReview, Branch source, Branch target) {
```

Comment:
To make it consistent and return conceptIds as Long please.

Reply:
Thanks, I have done this.